Directory.Build.props

```diff
@@ -46,7 +46,7 @@
   </PropertyGroup>
 
   <ItemGroup Label="NuGet" Condition="'$(MSBuildProjectExtension)' == '.vcxproj'">
-    <ProjectCapability Include="PackageReferences" />
+    <ProjectCapability Include="PackageReferences" Condition="@(PackageReference->Count()) &gt; 0" />
```

Comment:
I'm curious if this condition actually does anything, since this props file is likely to be included before any PackageReference item is actually defined

Reply:
Good point. I could and should have tested this using exactly the same technique.

It may be possible to embed this logic into a specific build phase.

Reply:
That said, the fact that both manual and CI keep working suggest this either:

A. Succeeds (maybe there's an initial item eval phase before props are parsed).
B. Is not needed to be explicitly declared anymore to support pkg refs.